src/opnsense/service/templates/OPNsense/IDS/suricata.yaml

```diff
@@ -444,7 +444,7 @@ outputs:
   - syslog:
       enabled: {% if helpers.empty('OPNsense.IDS.general.syslog') %}no{% else %}yes{% endif %}
 
-      # reported identity to syslog. If ommited the program name (usually
+      # reported identity to syslog. If omitted the program name (usually
```

Comment:
can merge this but should be reported upstream as this file is a direct copy from suricata

Reply:
Added https://github.com/OISF/suricata to my [list](https://github.com/stars/jsoref/lists/to-check-spelling), but, it's ever growing. It may take a bit for me to get to it.